sdcm/utils/s3_remote_uploader.py

```diff
@@ -71,7 +71,7 @@ def get_dir_size_kb(session, files):
     size = get_dir_size_kb(session, files)
     if size > max_size_gb:
         LOGGER.warning("Skipping upload '%s' directory to S3 due its size: %s GB", files, size)
-        return []
+        return ""
```

Comment:
why not to return `None`?

Reply:
Because function returns string. No need to switch to `None` and may introduce some errors.